File: 657-机器人能否返回原点.py
class Solution:
    def judgeCircle(self, moves: str) -> bool:
        x=0
        y=0
        for i in range(len(moves)):
            if moves[i] == 'L':
                x-=1
            if moves[i] == 'R':
                x+=1
            if moves[i] == 'U':
                y+=1
            if moves[i] == 'D':
                y-=1
        return x==0 and y==0

File: 657-机器人能否返回原点.cpp
class Solution {
public:
    bool judgeCircle(string moves) {
        int lens=moves.size();
        int x=0,y=0;
        for(int i=0;i<lens;i++)
        {
            if(moves[i]=='U')
                y++;
            if(moves[i]=='D')
                y--;
            if(moves[i]=='L')
                x--;
            if(moves[i]=='R')
                x++;
        }
        return x==0&&y==0;
    }
};
